Jenkinsfile

```diff
@@ -26,11 +26,10 @@ pipeline {
           def random = (new Random()).nextInt(10000000)
           def networkName = "build-" + random
           def seleniumName = "selenium-" + random
-          def ivyName = "ivy-" + random
           sh "docker network create ${networkName}"
           try {
             docker.image("selenium/standalone-firefox:4").withRun("-e START_XVFB=false --shm-size=2g --name ${seleniumName} --network ${networkName}") {
-              docker.build('maven-build').inside("--name ${ivyName} --network ${networkName}") {
```

Comment:
We don't need container name anymore?

Reply:
this repo does not contain any web tests pointing to the container itself